### DISCUSSION TITLE
Zigbee2mqtt

### DISCUSSION THREAD

**Discussion by angelo199045** [indomus/forum#2992] [ZigBee, Z-Wave, Thread, Matter]
Salve a tutti sono angelo e ho un problemino ho istallato mqtt e poi ho istallato zigbee2mqtt e lo configurato ora appena entro e faccio la ricerca dei dispositivi  mi va in errore quale può essere il problema???
![Screenshot_20240319_201420_Home Assistant](https://github.com/indomus/forum/assets/133518911/e67eb0cb-d033-4f79-9ba0-1f7b5faf8341)

**Comment by Scialla** [comment]
Touchlink? Sicuro di avere capito cosa stai facendo?
https://www.zigbee2mqtt.io/guide/usage/touchlink.html

Cosa stai cercando? Se devi associare dispositivi zigbee devi fare il pair

Magari indica anche che coordinator hai e cosa stai cercando di fare...

**Reply by angelo199045** [reply]
Allora scusate ho home assistant e devo aggiungere dei dispositivi zigbee che cosa devo fare???

**Reply by snakuzzo** [reply]
E' la prima volta che leggo una domanda del genere.
Ma poi scusa...te l'ha appena scritto @Scialla ... non hai letto ? Devi cliccare su PERMIT JOIN e poi mettere in modalità di pairing il device che vuoi collegare.
Touchlink serve ad altro!

**Reply by angelo199045** [reply]
E da poco che ho istallato home assistant , cmq grazie proverò a fare così!!!
